tests/mqtt-frames-truncated/test.rules

```diff
@@ -0,0 +1,17 @@
+alert mqtt any any -> any any (msg:"mqtt truncated Frame 1"; frame:pdu; content:"|10 1c|"; startswith; sid:1;)
```

Comment:
nit: msg indicates all these are truncated frames which from the matches does not seem to be true?